package/kernel/mac80211/realtek.mk

```diff
@@ -175,6 +177,24 @@ define KernelPackage/rtl8xxxu/description
   Please report your results!
 endef
 
+define KernelPackage/rtw88/config
+	config PACKAGE_RTW88_DEBUG
+		bool "Realtek wireless debugging (rtw88)"
+		depends on PACKAGE_kmod-rtw88
+		default n
```

Comment:
`default n` is not needed it is already set to n by default

Reply:
Thanks for the hint.
Fixed :+1: